From 0dee82eb4f72da186747cb07e84dda5bc8c8aeac Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 4 Jan 2013 18:25:15 -0500
Subject: [PATCH] profiles: Fix srcdir != builddir

---
 data/profiles/Makefile.am |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/profiles/Makefile.am b/data/profiles/Makefile.am
index 3af71d1..fd442fe 100644
--- a/data/profiles/Makefile.am
+++ b/data/profiles/Makefile.am
@@ -207,11 +207,11 @@ x11-colors.icc:
 		create-named-color srgb "X11" ""	\
 		--metadata "License=$(license)"		\
 		--description "X11 Colors"		\
-		./x11-colors.csv
+		$(srcdir)/x11-colors.csv
 crayons.icc:
 	$(top_builddir)/client/cd-create-profile --output $@ \
 		create-named-color lab "Crayon" ""	\
-		./crayons.csv				\
+		$(srcdir)/crayons.csv				\
 		--metadata "License=$(license)"		\
 		--description "Crayon Colors"
 bluish.icc:
-- 
1.7.1

