From 1ce22202a351f71b150cec716eafd9044caa81bf Mon Sep 17 00:00:00 2001
From: Vadim Rutkovsky <vrutkovs@redhat.com>
Date: Thu, 24 Apr 2014 12:16:57 +0200
Subject: [PATCH] tests: report in html also

---
 m4/behave-installed-tests.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/behave-installed-tests.m4 b/m4/behave-installed-tests.m4
index 386fe5f..baf01bd 100644
--- a/m4/behave-installed-tests.m4
+++ b/m4/behave-installed-tests.m4
@@ -106,7 +106,7 @@ installed-tests-data-hook:
 	@for test in $(INSTALLED_TESTS); do							\
 	    echo "Installing $$test.test to $(META_DIRECTORY)";					\
 	    echo m4_escape([[Test]]) > $(META_DIRECTORY)/$$test.test;				\
-	    echo "Exec=behave $(pkglibexecdir)/installed-tests -t $$test -k -f plain"	\
+	    echo "Exec=behave $(pkglibexecdir)/installed-tests -t $$test -k -t html -o $$test.html -f plain"	\
 	                                           >> $(META_DIRECTORY)/$$test.test;		\
 	    echo "Type=$(INSTALLED_TESTS_TYPE)" >> $(META_DIRECTORY)/$$test.test;		\
 	done
-- 
1.9.0

