From 41c374d6201782a5245ac20d31fff5a61779a122 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 25 May 2012 18:20:45 -0400
Subject: [PATCH] build: Note we don't build when srcdir != builddir

---
 configure.in |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/configure.in b/configure.in
index 4c92632..e43e16a 100644
--- a/configure.in
+++ b/configure.in
@@ -6,6 +6,10 @@ AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CANONICAL_HOST
 
+dnl We don't build with srcdir != builddir at present; the Python
+dnl bindings fail
+echo \#buildapi-variable-no-builddir > /dev/null
+
 dnl LT Version numbers, remember to change them *before* a release.
 dnl   (Interfaces removed:    CURRENT++, AGE=0, REVISION=0)
 dnl   (Interfaces added:      CURRENT++, AGE++, REVISION=0)
-- 
1.7.7.6

