From 93971f58aa317bc5263783ee74ad0679cb8267bc Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sun, 24 Feb 2013 05:12:33 -0500
Subject: [PATCH] build: Look for generated file in builddir, not srcdir

---
 libmm-glib/generated/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libmm-glib/generated/Makefile.am b/libmm-glib/generated/Makefile.am
index 16dd413..c0429b1 100644
--- a/libmm-glib/generated/Makefile.am
+++ b/libmm-glib/generated/Makefile.am
@@ -66,7 +66,7 @@ mm-errors-types.c: Makefile.am $(top_srcdir)/include/ModemManager-errors.h $(top
 		--template $(top_srcdir)/build-aux/mm-errors-template.c \
 		$(top_srcdir)/include/ModemManager-errors.h > $@
 
-mm-errors-quarks.c: Makefile.am $(top_srcdir)/include/ModemManager-errors.h $(top_srcdir)/build-aux/mm-errors-quarks-template.c $(top_srcdir)/include/ModemManager-names.h mm-errors-types.h
+mm-errors-quarks.c: Makefile.am $(top_srcdir)/include/ModemManager-errors.h $(top_srcdir)/build-aux/mm-errors-quarks-template.c $(top_builddir)/include/ModemManager-names.h mm-errors-types.h
 	$(AM_V_GEN) $(GLIB_MKENUMS) \
 		--fhead "#include <ModemManager.h>\n#include \"mm-errors-types.h\"\n" \
 		--template $(top_srcdir)/build-aux/mm-errors-quarks-template.c \
-- 
1.7.1

