From b6516bfd5db25e4d133133b79b3dec3ab07b19f7 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sun, 23 Sep 2012 15:25:38 -0400
Subject: [PATCH] Disable parallel builds for now

---
 GNUmakefile.am | 5 +++++
 GNUmakefile.in | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/GNUmakefile.am b/GNUmakefile.am
index 8eeb9ff..26efe9f 100644
--- a/GNUmakefile.am
+++ b/GNUmakefile.am
@@ -349,3 +349,8 @@ if ENABLE_WEBKIT2
 	@-cp ./Documentation/webkit2gtk/tmpl/*.sgml $(distdir)/Documentation/webkit2gtk/tmpl
 	@cp ./Documentation/webkit2gtk/html/* $(distdir)/Documentation/webkit2gtk/html
 endif
+
+# Parallel builds are broken at the moment:
+# https://bugs.webkit.org/show_bug.cgi?id=79498
+# http://savannah.gnu.org/bugs/?30653
+.NOTPARALLEL:
diff --git a/GNUmakefile.in b/GNUmakefile.in
index 64d602d..06214f2 100644
--- a/GNUmakefile.in
+++ b/GNUmakefile.in
@@ -80638,3 +80638,8 @@ doc-dist-hook: docs-build.stamp
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
+
+# Parallel builds are broken at the moment:
+# https://bugs.webkit.org/show_bug.cgi?id=79498
+# http://savannah.gnu.org/bugs/?30653
+.NOTPARALLEL:
-- 
1.7.11.4

