From 520733c4a2610dea109f846ecccbaf2b4957b8f7 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sat, 17 Aug 2013 17:19:35 -0400
Subject: [PATCH] autogen.sh: Honor NOCONFIGURE=1

---
 autogen.sh |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 4e8b11b..dbcce6d 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -9,4 +9,6 @@ cd $srcdir
 autoreconf --force -v --install || exit 1
 cd $ORIGDIR || exit $?
 
-$srcdir/configure --enable-maintainer-mode "$@"
+if test -z "$NOCONFIGURE"; then
+    exec $srcdir/configure --enable-maintainer-mode "$@"
+fi
-- 
1.7.1

